data/en/mbs_0117.json

```diff
@@ -0,0 +1,424 @@
+                                            "list": [
+                                                "VAT",
+                                                "income from the sales of fixed capital assets",
+                                                "grants, funding, donations etc",
+                                                "insurance claims",
+                                                "interest received"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                {
+                                    "question": "Changes in turnover",
+                                    "content": [{
+                                        "description": "Include:",
+                                        "list": [
+                                            "change in level of business activity",
+                                            "maintenance/shutdowns",
+                                            "special/calendar events",
+                                            "weather",
+                                            "price effects",
+                                            "currency effects (increase/decrease in the currency value)"
+                                        ]
+                                    }]
+                                },
+                                {
+                                    "question": "Value of grants, funding, donations and investment income",
+                                    "content": [{
+                                        "description": "Exclude:",
+                                        "list": [
+                                            "VAT"
+                                        ]
+                                    }]
+                                }
+                            ]
+                        },
+                        "secondary_content": [{
+                            "id": "how-we-use-your-data",
+                            "title": "How we use your data",
+                            "content": [{
+                                "list": [
+                                    "You cannot appeal your selection. Your business was selected to give us a comprehensive view of the UK economy.",
+                                    "The information you provide contributes to <a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/economy/grossdomesticproductgdp'>Gross Domestic Product (GDP).</a>"
+                                ]
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "reporting-period-block",
+                        "type": "Question",
+                        "questions": [{
+                            "description": "",
```

Comment:
Empty descriptions should not be a thing, if we're not using them remove them and same with labels

Reply:
Also title is not mandatory in preview_content but it is mandatory in groups